src/test/java/shop/RealItemTest.java

```diff
@@ -0,0 +1,32 @@
+package shop;
+
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RealItemTest {
+    RealItem testRealItem;
+    private static final double WEIGTH = 100;
+
+    @BeforeEach
+    void setUp() {
+        testRealItem = new RealItem();
+    }
+
+    @Test
+    @Tag("Real_item_set_and_get_weigth")
+    @Tag ("Smoke")
+    @DisplayName("Set weigth and get weigth")
+    void setWeightAndGetWeigth() {
+
+        double weigthBefore = testRealItem.getWeight();
+        testRealItem.setWeight(WEIGTH);
+        double weigthAfter = testRealItem.getWeight();
+
+        assertAll("Weigth",
```

Comment:
RealItem class is designed as simple model class, which doesn't have any business logic. It doesn't make sense to develop tests for getters/setters. The only method that can be covered with test - toString() method (because class overrides it).
This task has 2 correct resolutions:
- don't write tests at all for this class
- write test for toString method

Reply:
I decided to write a test for toString method.
commit 487b7687d5950577df244c3e7daf072f51388f44
commit 6a037f7ebd2c4bdea4f5dba81c481c592a01c609